bundle/definition/validators.go

```diff
@@ -1,45 +1,54 @@
 package definition
 
 import (
+	"context"
 	"encoding/base64"
 	"fmt"
 
+	"github.com/qri-io/jsonpointer"
 	"github.com/qri-io/jsonschema"
 )
 
 // ContentEncoding represents a "custom" Schema property
 type ContentEncoding string
 
 // NewContentEncoding allocates a new ContentEncoding validator
-func NewContentEncoding() jsonschema.Validator {
+func NewContentEncoding() jsonschema.Keyword {
 	return new(ContentEncoding)
 }
 
-// Validate implements the Validator interface for ContentEncoding
-// which, as of writing, isn't included by default in the jsonschema library we consume
-func (c ContentEncoding) Validate(propPath string, data interface{}, errs *[]jsonschema.ValError) {
+func (c ContentEncoding) Validate(propPath string, data interface{}, errs *[]jsonschema.KeyError) {}
+
+func (c ContentEncoding) ValidateKeyword(ctx context.Context, currentState *jsonschema.ValidationState, data interface{}) {
 	if obj, ok := data.(string); ok {
 		switch c {
 		case "base64":
 			_, err := base64.StdEncoding.DecodeString(obj)
 			if err != nil {
-				jsonschema.AddError(errs, propPath, data, fmt.Sprintf("invalid %s value: %s", c, obj))
+				currentState.AddError(data, fmt.Sprintf("invalid %s value: %s", c, obj))
 			}
 		// Add validation support for other encodings as needed
 		// See https://json-schema.org/latest/json-schema-validation.html#rfc.section.8.3
 		default:
-			jsonschema.AddError(errs, propPath, data, fmt.Sprintf("unsupported or invalid contentEncoding type of %s", c))
+			currentState.AddError(data, fmt.Sprintf("unsupported or invalid contentEncoding type of %s", c))
 		}
 	}
 }
 
+func (c ContentEncoding) Register(uri string, registry *jsonschema.SchemaRegistry) {}
```

Comment:
Looks like this method as well as `Resolve` below can/should be no-ops but needed to implement an interface?  Or, will there be scenarios where we'd need to add logic for these?